src/web/releases.rs

```diff
@@ -82,13 +82,14 @@ pub(crate) fn get_releases(conn: &mut Client, page: i64, limit: i64, order: Orde
             releases.target_name,
             releases.release_time,
             releases.rustdoc_status,
-            crates.github_stars
+            github_repos.stars
         FROM crates
         INNER JOIN releases ON crates.id = releases.crate_id
+        LEFT JOIN github_repos ON releases.github_repo = github_repos.id
         WHERE
             ((NOT $3) OR (releases.build_status = FALSE AND releases.is_library = TRUE))
             AND crates.latest_version_id = releases.id
-        ORDER BY {} DESC
+        ORDER BY {} DESC NULLS LAST
```

Comment:
Can you add a test that crates without github metadata are shown last on the 'releases by stars' page?